src/app/user/(logged-in)/layout.tsx

```diff
@@ -25,6 +25,12 @@ export default async function Layout(props: { children: React.ReactNode }) {
       iconSrc: IMG_PATH.SIDEBAR_USER,
       iconLightSrc: IMG_PATH.SIDEBAR_USER_LIGHT,
     },
+    {
+      label: "Info",
+      href: RoutePath.USER_INFO,
+      iconSrc: IMG_PATH.SIDE_BAR_INFO,
+      iconLightSrc: IMG_PATH.SIDE_BAR_INFO_LIGHT,
+    },
     {
       label: "Criteria",
       href: RoutePath.USER_CRITERIA,
```

Comment:
The Criteria and Process options can be removed. They are replaced by Info.

Reply:
Bump.

Reply:
Done! 9adb788fd8d1d822aa5fd078179a337658403588

Reply:
Process also. (Was not able to highlight it for some reason 🙏)

Thanks!

Reply:
Done! 4fbc07ba293a22f616be32dbe51528cd19f3c574